File: app/src/main/java/pt/estga/spotme/ui/parking/ParkingFormViewModel.java
package pt.estga.spotme.ui.parking;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import pt.estga.spotme.entities.Parking;

public class ParkingFormViewModel extends ViewModel {
    private final MutableLiveData<Parking> parkingLiveData = new MutableLiveData<>();

    public LiveData<Parking> getParking() {
        return parkingLiveData;
    }

    public void setParking(Parking parking) {
        parkingLiveData.setValue(parking);
    }

}

File: app/src/main/java/pt/estga/spotme/ui/parking/ParkingFormViewModel.kt
package pt.estga.spotme.ui.parking

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import pt.estga.spotme.entities.Parking

class ParkingFormViewModel : ViewModel() {
    private val parkingLiveData = MutableLiveData<Parking?>()

    val parking: LiveData<Parking?>
        get() = parkingLiveData

    fun setParking(parking: Parking?) {
        parkingLiveData.value = parking
    }
}